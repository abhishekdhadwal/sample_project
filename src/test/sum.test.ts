
import sum from './sum';


describe("test add function", () => {
  it("should return 15 for add(10,5)", () => {
    let result = sum(10,5); 
    expect(result).toBe(14);
    expect(result).toMatchSnapshot();
  });
  it("should return 9 for add(3,6)", () => {
    let response = sum(3,6);
    expect(response).toBe(9);
    expect(response).toMatchSnapshot();
  });

})


