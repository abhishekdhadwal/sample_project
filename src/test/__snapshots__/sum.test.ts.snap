// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test add function should return 5 for add(2,3) 1`] = `5`;

exports[`test add function should return 9 for add(3,6) 1`] = `9`;

exports[`test add function should return 15 for add(10,5) 1`] = `15`;
